code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return TRUE
 
 	return FALSE
+	
+/mob/dead/observer/incapacitated()
+	return TRUE

```

Comment:
Well the thing is that instead of doing the long string of stat, restrained, lying checks for verbs (such as toggling water tanks) we now just use incapacitated(). However, this means that ghosts are able to use them.

I don't think there's any situation where a ghost should be able to do something that has an incapacitated() check.